packages/schemas/src/gen/index.ts

```diff
@@ -82,6 +84,7 @@ const generate = async () => {
                 return {
                   name,
                   type: primitiveType,
```

Comment:
Why `primitiveType`? Will it always be like `string`, `number`, `boolean`?

Reply:
if it is a primitive type, it will 'string' | 'number' | 'boolean' | 'Record<string, unknown>'.